mixed-content/generic/mixed-content-test-case.js

```diff
@@ -0,0 +1,224 @@
+function MixedContentTestCase(scenario, description, sanityChecker) {
+  var insecureProtocol = "http:"
```

Comment:
I was using this format so it's comparable to location.protocol - comparing is easier. Any idea why location.protocol is using a colon?
 
Refactored.